### DISCUSSION TITLE
Changes in ecFSEOF: GECKO 2.0 -> 3.0

### DISCUSSION THREAD

**Discussion by JBiaggi** [SysBioChalmers/GECKO#355] [Q&A] [answered]
Hello! I've been using your ecFSEOF for GECKO 3.0, and my results seem counterintuitive. In a nutshell, I'm trying to get targets to increase lipid production, but the k-scores for enzymes related to lipid production are all below 0.5. When investigating this results, I noticed that the function 'getFluxTarget' has changed compared to 'simulateGrowth' (its equivalent in GECKO 2.0) in a way that the former first minimizes the protein pool and than maximizes the enforced objective, whereas the latter does it the other way around. Could you please explain the rationale that motivated this change?

**Comment by ae-tafur** [comment, marked as answer]
Hi, 

The enforce objective in both GECKO version are first maximize (biomass), then, the growth rate reached it is fixed. In GECKO 3, we first minimize protein pool, assuming that the cell will use protein only to growth rather to produce a by-product. Because of that, after fixing growth rate, protein pool is minimized and afterwards target product in maximized to get the flux distribution. 

Finally,  are you working with yeast ? have you try ecFSEOF under GECKO 2 version ? if yes, can you share how different the results are ?

**Reply by JBiaggi** [reply]
Hi!

Thanks for you answer! I'm working with an ecModel of _Yarrowia lipolytica_ that I generated myself using GECKO 3.0. It is in my plans to test the ecFSEOF from GECKO 2 as well, but I haven't done that yet. When I do, I'm pretty sure I can share the results with you.
